engines/registration_sharing/app/controllers/registration_sharing/rmt_to_rmt_controller.rb

```diff
@@ -10,6 +10,13 @@ def create
         system = System.find_or_create_by(login: params[:login])
         system.update(system_params)
 
+        # TODO: remove this block when proxy_byos column gets dropped
+        if !params.key?(:proxy_byos_mode) && system.attribute_names.include?('proxy_byos_mode')
```

Comment:
because this RMT version includes `proxy_byos_mode`

`proxy_byos` is included in 2.17 and 2.19

so if a system gets registered in 2.17 and shared the info to a 2.19, we want `proxy_byos_mode` to be populated as well

Reply:
My concern is that the code checks that `proxy_byos_mode` param is _not present_ but does not check that `proxy_byos` param _is present_, even though that's the param used in subsequent code.

Reply:
I understand that right now in our codebase those things are logically equal, but it won't always be the case, so IMO it makes more sense to test for the thing you're going to use. Am I missing something?

Reply:
`proxy_byos _is_ present, we have not removed that column/param neither before this version nor in this version, why check something we know it is there ?

Reply:
said with different words, that handling needs to happen if `proxy_byos_mode` is not present in the params and the RMT server has the _new_ param, otherwise, nothing to do

Reply:
Okay, acknowledging that we're just going to live with this code smell for now.